src/scancode/api.py

```diff
@@ -183,12 +183,12 @@ def get_file_infos(location, as_list=True):
     infos['mime_type'] = is_file and T.mimetype_file or None
     infos['file_type'] = is_file and T.filetype_file or None
     infos['programming_language'] = is_file and T.programming_language or None
-    infos['is_binary'] = is_file and T.is_binary or None
-    infos['is_text'] = is_file and T.is_text or None
-    infos['is_archive'] = is_file and T.is_archive or None
-    infos['is_media'] = is_file and T.is_media or None
-    infos['is_source'] = is_file and T.is_source or None
-    infos['is_script'] = is_file and T.is_script or None
+    infos['is_binary'] = bool(is_file and T.is_binary)
```

Comment:
`bool` is not absolutely necessary, but I would rather be explicit on the api side and not rely of the values returned as the `typecode` implementation could evolve. Internally `typecode` uses to emulated tri-booleans (e.g. None, True or False) and this is not super clean and will need some refactoring.